src/angular/controllers/map.controller.js

```diff
@@ -3,6 +3,20 @@ var controllers = angular.module('controllers');
 controllers.controller('MapCtrl', ['$scope', '$rootScope', '$location', '$translate', 'Search', function ($scope, $rootScope, $location, $translate, Search) {
     var mapCtrl = this;
 
+    mapCtrl.onRegionClick = function(event) {
```

Comment:
I think we can delete this, we're not going to allow clicking on regions anymore.